R/eval.R

```diff
@@ -96,12 +96,9 @@ evaluate <- function(input,
     if (length(dev.list()) < devn) dev.set(dev)
     devn <- length(dev.list())
 
-    expr <- parsed$expr[[i]]
-    if (!is.null(expr))
-      expr <- as.expression(expr)
```

Comment:
So I understand we don't try to coerce anymore, and thown an error instead later if `stopifnot(is.expression(exprs))`, because we do use `expression()` and not `NULL` as default in `parse_all()`, right ?

Probably find. Just being sure I am following correctly.

Reply:
Exactly! Or another way of saying it is that we ensure that `parse_all()` always returns expressions, so we don't have to coerce here.